<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>765</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1174</width>
    <height>765</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Roads To Castle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1174</width>
     <height>725</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1174</width>
      <height>741</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1174</width>
      <height>741</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/backgrounds/second_window.png);
border-image: url(:/new/backgrounds/second_window.png);</string>
    </property>
    <widget class="QWidget" name="startpage">
     <property name="styleSheet">
      <string notr="true">image: url(:/new/backgrounds/ROAD.jpg);</string>
     </property>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>620</y>
        <width>341</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/button/button.jpg);
border-image: url(:/button/button.jpg);

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="secondpage">
     <widget class="QPushButton" name="browse">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>80</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/button/browsebutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="music">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>147</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/button/musicbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/button/exitbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="output">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/backgrounds/output_screen.png);</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>680</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/button/outputbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="intextEdit">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>110</y>
        <width>461</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/backgrounds/base.png);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="outtextEdit">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>116</y>
        <width>451</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/backgrounds/base.png);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/button/back.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionTurn_music_off"/>
    <addaction name="actionTurn_music_on"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menusettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionf">
   <property name="text">
    <string>Turn music off</string>
   </property>
  </action>
  <action name="actionTurn_music_off">
   <property name="text">
    <string>Turn music off</string>
   </property>
  </action>
  <action name="actionTurn_music_on">
   <property name="text">
    <string>Turn music on</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionexit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
